ddtrace/internal/writer.py

```diff
@@ -43,6 +42,12 @@
 from .sma import SimpleMovingAverage
 
 
+should_start_remoteconfig = config._appsec_enabled or asbool(os.environ.get("DD_REMOTE_CONFIGURATION_ENABLED", "true"))
```

Comment:
I don't love this, and am amenable to coalescing these in config if that's preferable.